game/selenium_tests/base.py

```
@@ -0,0 +1,92 @@
+        chrome_options.add_argument("--disable-dev-shm-usage")
+        chrome_options.add_argument("--disable-gpu")
+        chrome_options.add_argument("--window-size=1920,1080")
+
+        try:
+            cls.driver = webdriver.Chrome(options=chrome_options)
+            log_ok("Chrome WebDriver initialized")
+        except Exception as e:
+            log_fail(f"Failed to initialize Chrome WebDriver: {e}")
+            raise RuntimeError(
+                f"Failed to initialize Chrome WebDriver: {e}"
+            ) from e
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)